src/views/Restoration/RestorationBusinessName.vue

```diff
@@ -165,43 +177,36 @@ export default class RestorationBusinessName extends Vue {
   @Getter getShowErrors!: boolean
   @Getter getBusinessContact!: ContactPointIF
   @Getter getFolioNumber!: string
-
+  @Getter isRestorationTypeValid!: boolean
   @Action setBusinessContact!: ActionBindingIF
   @Action setCooperativeType!: ActionBindingIF
   @Action setFolioNumber!: ActionBindingIF
   @Action setOfficeAddresses!: ActionBindingIF
   @Action setDefineCompanyStepValidity!: ActionBindingIF
   @Action setIgnoreChanges!: ActionBindingIF
-
```

Comment:
Again, our non-official code style is to leave whitespace between separate blocks. Please restore it.